src/isIterable.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Checks whether the passed value is iterable.
+ *
+ * @func isIterable
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.18.0|v2.18.0}
+ * @category Type
+ * @sig * -> Iterable
+ * @param {*} val The value to test
+ * @return {boolean}
+ * @example
+ *
+ * RA.isIterable(['arrays', 'are', 'iterable']); //=> true
```

Comment:
Let's limit example to 6 values max here. Having all variations in example is not necessary.

Reply:
I certainly understand why you would request that.  Please tell me which 6 you would like to keep.  I did remove a few of them. 

To the uninitiated, some people may be surprised that you can iterate over a string in a for loop, or that you can iterate over an empty string (which is otherwise sometimes falsey).  Similarly, I think it's useful to know that functions are not iterable but the return of a generator _is_.   Just the same, I think some people might expect that a WeakMap or WeakSet (without deeply understanding the collections themselves and how they work) would be iterable.  Not to mention, I left out from the example that you can create your own iterable (although it's in the tests) and I think that might be something worth including.  This is a somewhat tricky topic and I feel like it's best to be clear with how the function works.

Furthermore, I was wondering if you would have any opinions about how I handled checking for the presence of `Symbol`.  As it relates to this topic: perhaps we might put a mention in the example or elsewhere as to what happens if your browser doesn't support it.

All up to you :)

Reply:
> Furthermore, I was wondering if you would have any opinions about how I handled checking for the presence of Symbol. As it relates to this topic: perhaps we might put a mention in the example or elsewhere as to what happens if your browser doesn't support it.

I provided my opinion about implementation in another comment. IMHO we should return false when there is no support for iterables, because non of the objects in such environment is iterable.

Reply:
> To the uninitiated, some people may be surprised that you can iterate over a string in a for loop, or that you can iterate over an empty string (which is otherwise sometimes falsey). Similarly, I think it's useful to know that functions are not iterable but the return of a generator is. Just the same, I think some people might expect that a WeakMap or WeakSet (without deeply understanding the collections themselves and how they work) would be iterable. Not to mention, I left out from the example that you can create your own iterable (although it's in the tests) and I think that might be something worth including. This is a somewhat tricky topic and I feel like it's best to be clear with how the function works.

That is why we have tests or people can open the browser console on our [documentation](https://char0n.github.io/ramda-adjunct/2.17.0/) and write `RA.isIterable(3)` to see the result. IMHO 6 cases suffice, like showing array, string and generator returns true; null, undefined and numbers returns false...

Reply:
ok. updating now.